src/main.py

```diff
@@ -0,0 +1,76 @@
+import argparse
+
+import config
+import rds
+import tf
+import utils
+
+LOGGER = utils.setup_custom_logger("root")
+
+
+def main(command_line=None):
+    parser = argparse.ArgumentParser(description="Restore RDS snapshot")
+    parser.add_argument("-c", "--config", required=True)
+    parser.add_argument("-t", "--tfstate")
```

Comment:
Love the usage of the ArgumentParser lib here.  Should `--tfstate` be required too?

Reply:
Nope, its optional